README.md

```diff
@@ -130,9 +133,11 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Tech101](https://www.tech101.co.uk/) | Kamaladi, Kathmandu | Web design, web development, graphics design and digital marketing
 - [ThemeGrill](https://themegrill.com/) | Basundhara, Kathmandu | Free and Premium WordPress Themes Store
 - [Three Monks](http://www.threemonk.com/) | New Baneshwor, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
+- [UBA Solutions](http://www.uba-solutions.com/) | Thadodhunga, Lalitpur | Software Research and Development
 - [Unelma Platforms](http://unelmaplatforms.com/) | Ratnanagar, Chitwan | Unelma Platforms is a provider of software development services for higher-education, healthcare and business markets
 - [UniAxilSoftwares](http://uniaxialsoftwares.com/) | Anamnagar, Kathmandu | Innovation - for real world problems
 - [Unifun](https://unifun.com/) | Nagpokhari, Kathmandu | Unifun is a provider of high-class IT products for Mobile Operators
+- [Unlimited Technology Pvt Ltd](http://unlimit.com) | Unlimited Building, Khichapokhari | expertise in business software, custom software, business intelligence, document management/collaboration, mobile and infrastructure solutions
```

Comment:
Can you make this `Khichapokhari, Kathmandu`. 

Also, can you start the description with a capital letter. It should be:

```diff
- expertise
+ Expertise
```